features/step_definitions/portfolio.rb

```diff
@@ -110,6 +110,23 @@ def getUUID(itemName)
   click_on item
 end
 
+
+Quand("on met {string} dans la barre de recherche et on clique sur {string}") do |value1, value2|
+    #fill_in "Rechercher", with: 'value1', visible: false
+
+    #within("form-control#Rechercher") do
+    #fill_in "Rechercher", :with => @visitor[:value2]
```

Comment:
One of the criterium in the check list is "(being) useful (it would not work if removed)".
This applies especially to commented out dead code (see [Cruft (jargon)](https://en.wikipedia.org/wiki/Cruft)).

Please remove every unused code line.